CIME/provenance.py

```diff
@@ -54,8 +55,16 @@ def _extract_times(zipfiles, target_file):
                     target, the_time = items[1], items[-2]
                     contents += "{} {}\n".format(target, the_time)
 
+        stat, output, _ = run_cmd("zgrep -E '^user [0-9.]+$' {}".format(zipfile))
```

Comment:
I think we should use the "real" time reported by the time command to account for all the I/O and system/kernel overheads involved.

Reply:
I think you're right.